src/main/java/frc/robot/subsystems/PoseEstimationSubsystem.java

```diff
@@ -88,7 +88,7 @@ public Pose2d getPose() {
     private void updateEstimation() {
         estimator.update(drivetrain.getGyro(), drivetrain.getModulePositions());
         if (visionSubsystem.seeingAnything()) {
-        //    estimator.addVisionMeasurement(visionSubsystem.getRobotPose(), Timer.getFPGATimestamp());
+           estimator.addVisionMeasurement(visionSubsystem.getRobotPose(), Timer.getFPGATimestamp());
```

Comment:
This is very risky to uncomment unless we've validated it on compbot, on an actual real field